src/dotnet-svcutil/lib/src/Shared/MSBuildProj.cs

```diff
@@ -159,6 +160,13 @@ public static async Task<MSBuildProj> ParseAsync(string projectText, string proj
                     var targetFramework = targetFrameworkElements.Last().Value.Trim();
                     if (!string.IsNullOrWhiteSpace(targetFramework))
                     {
+                        Regex re = new Regex(@"\w+\d+\.\d+");
```

Comment:
What is the reason of using regex here?  What about the approach with just comparing with "net5.0-windows*" approach?

Reply:
The possible target framework format examples are: netcoreapp3.1, net5.0, net5.0-windows, the regex here is to truncate the extra characters after version number, so for both net5.0 and net5.0-windows, or other target framework format like "net5.0***" appear in the furture, we will get net5.0 for the internal starter project to generate service proxy code.

Reply:
> What is the reason of using regex here? What about the approach with just comparing with "net5.0-windows*" approach?

I have removed regex code and use "net5.0-windows" approach, could you please help review the update?

Reply:
I've modified the code according to the meeting, could you please help review the update?